channels/ir.m3u

```diff
@@ -211,3 +211,5 @@ http://163.172.138.186/hls/stream.m3u8
 https://voa-lh.akamaihd.net/i/voapnn_7@72817/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="YourTime TV" tvg-language="Persian" tvg-logo="https://yourtime.tv/img/utv-logo.png" group-title="Entertainment",YourTime TV
 https://hls.yourtime.live/hls/stream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="IRIB Nasim" tvg-language="Persian" tvg-logo="https://upload.wikimedia.org/wikipedia/en/a/a0/IRIB_Nasim_Logo.png" group-title="Entertainment",IRIB Nasim
+https://cdn1live.irib.ir/live-channels/smil:nasim/playlist.m3u8?s=xfzwmaVbFGKsE7GxWxNrKw&t=1594967100
```

Comment:
Link no longer works. Tested in Elmedia Video Player, VLC and streamtest.in

Reply:
Hi, I found a working link:
https://live.farsiland.com/nasim/stream.m3u8